### PR TITLE
image browser component added

### PR DESCRIPTION
**Pull request by pragati-9bit**

### CODE REVIEW

File: src/js/components/imageBrowser/imageBrowser.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+          </div>
+        </div>
+      </div>
+        `;
+    // Function to load related media
+    /**
+     * loadRelatedMedia
+     */
+    function loadRelatedMedia() {
+      const relatedMediaContainer = document.getElementById('relatedMedia');
```
Comment (rajeev-9bit) [inline src/js/components/imageBrowser/imageBrowser.js:42]: avoid using `document` to find internal elements

File: src/js/components/imageBrowser/imageBrowser.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+        // Add hover listener to change main image on hover
+        img.addEventListener('mouseover', () => {
+          document.getElementById('mainImage').src = mediaURL;
+        });
+        imgContainer.appendChild(img);
+        relatedMediaContainer.appendChild(imgContainer);
+      });
+    }
+    // Load related media on page load
+    window.onload = loadRelatedMedia;
```
Comment (rajeev-9bit) [inline src/js/components/imageBrowser/imageBrowser.js:62]: try to find alternate of `window onload`. 
See https://developer.mozilla.org/en-US/docs/Web/API/Web_components/Using_custom_elements

**Review by rajeev-9bit — COMMENTED** [review]
Please address comments

File: src/js/components/imageBrowser/imageBrowser.js (hunk 2 above)
Comment (pragati-9bit) [reply src/js/components/imageBrowser/imageBrowser.js]: Sir, I have resolved both issues. You can review the code as I have already pushed it.

**Review by rajeev-9bit — APPROVED** [review]

**Pull request merged**